pydantic_ai_slim/pydantic_ai/models/test.py

```diff
@@ -79,6 +79,25 @@ class TestModel(Model):
     _model_name: str = field(default='test', repr=False)
     _system: str = field(default='test', repr=False)
 
+    def __init__(
```

Comment:
Should we add `init=False` to the dataclass decorator?

Reply:
Yes, ofc!